pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -49,11 +45,76 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  howard-hinnant-date = fetchFromGitHub {
+    owner = "HowardHinnant";
+    repo = "date";
+    rev = "v2.4.1";
+    sha256 = "sha256-BYL7wxsYRI45l8C3VwxYIIocn5TzJnBtU0UZ9pHwwZw=";
+  };
+
+  mp11 = fetchFromGitHub {
+    owner = "boostorg";
+    repo = "mp11";
+    rev = "boost-1.79.0";
+    sha256 = "sha256-ZxgPDLvpISrjpEHKpLGBowRKGfSwTf6TBfJD18yw+LM=";
+  };
+
+  safeint = fetchFromGitHub {
+    owner = "dcleblanc";
+    repo = "safeint";
+    rev = "ff15c6ada150a5018c5ef2172401cb4529eac9c0";
+    sha256 = "sha256-PK1ce4C0uCR4TzLFg+elZdSk5DdPCRhhwT3LvEwWnPU=";
+  };
+
+  pytorch_cpuinfo = fetchFromGitHub {
+    owner = "pytorch";
+    repo = "cpuinfo";
+    # There are no tags in the repository
+    rev = "5916273f79a21551890fd3d56fc5375a78d1598d";
+    sha256 = "sha256-nXBnloVTuB+AVX59VDU/Wc+Dsx94o92YQuHp3jowx2A=";
+  };
+
+  wil = fetchFromGitHub {
+    owner = "microsoft";
+    repo = "wil";
+    rev = "v1.0.230411.1";
+    sha256 = "sha256-5DBJDgwZljfiCFMPlPE9aZ41rKnEVzLUcxRTgxqPWHw=";
+  };
+
+  gsl = fetchFromGitHub {
+    owner = "microsoft";
+    repo = "GSL";
+    rev = "v4.0.0";
+    sha256 = "sha256-cXDFqt2KgMFGfdh6NGE+JmP4R0Wm9LNHM0eIblYe6zU=";
+  };
+
+  flatbuffers = fetchFromGitHub {
+    owner = "google";
+    repo = "flatbuffers";
+    rev = "v1.12.0";
+    sha256 = "sha256-L1B5Y/c897Jg9fGwT2J3+vaXsZ+lfXnskp8Gto1p/Tg=";
+  };
+
+  eigen = fetchFromGitLab {
+    owner = "libeigen";
+    repo = "eigen";
+    # This commit is not part of master or any release!
+    rev = "d10b27fe37736d2944630ecd7557cefa95cf87c9";
+    sha256 = "sha256-Lmco0s9gIm9sIw7lCr5Iewye3RmrHEE4HLfyzRkQCm0=";
+  };
+
+  gtest = fetchFromGitHub {
```

Comment:
https://github.com/NixOS/nixpkgs/pull/228209 will provide a new enough version of `gtest` in `nixpkgs`.

Reply:
FYI: The PR got reverted due to problems with a version bump of the lowest supported C++ standard in `gtest`.

Reply:
Please use override on our gtest.

Also this all should be moved to a let in to not accidentally create envs with the contents.